aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/CreateHandler.java

```diff
@@ -106,7 +108,15 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     return progress;
                 })
                 .then(progress -> addAssociatedRoles(proxy, rdsProxyClient, progress, progress.getResourceModel().getAssociatedRoles(), false))
-                .then(progress -> new ReadHandler().handleRequest(proxy, request, callbackContext, rdsProxyClient, ec2ProxyClient, logger));
+                .then(progress -> {
+                    model.setTags(Translator.translateTagsFromSdk(Tagging.translateTagsToSdk(allTags)));
```

Comment:
I think this might fail detecting drift in resource tags since we always set them from desired model.